sycl/test-e2e/EnqueueNativeCommand/custom-command-level-zero.cpp

```diff
@@ -0,0 +1,135 @@
+// RUN: %{build} %level_zero_options -o %t.out
+// RUN: %{run} %t.out
+
+// REQUIRES: level_zero, level_zero_dev_kit
+
+// TODO: Reenable, see https://github.com/intel/llvm/issues/14598
+// UNSUPPORTED: windows
+
+#include <level_zero/ze_api.h>
+
+#include <iostream>
+#include <sycl/backend.hpp>
+#include <sycl/detail/core.hpp>
+#include <sycl/interop_handle.hpp>
+
+using namespace sycl;
+using namespace sycl::access;
+
+static constexpr size_t BUFFER_SIZE = 1024;
+
+template <typename T> class Modifier;
+
+template <typename T> class Init;
+
+template <typename BufferT, typename ValueT>
+void checkBufferValues(BufferT Buffer, ValueT Value) {
+  auto Acc = Buffer.get_host_access();
+  for (size_t Idx = 0; Idx < Acc.get_count(); ++Idx) {
+    if (Acc[Idx] != Value) {
+      std::cerr << "buffer[" << Idx << "] = " << Acc[Idx]
+                << ", expected val = " << Value << '\n';
+      exit(1);
+    }
+  }
+}
+
+template <typename DataT>
+void copy(buffer<DataT, 1> &Src, buffer<DataT, 1> &Dst, queue &Q) {
+  Q.submit([&](handler &CGH) {
+    auto SrcA = Src.template get_access<mode::read>(CGH);
+    auto DstA = Dst.template get_access<mode::write>(CGH);
+
+    auto Func = [=](interop_handle IH) {
+      auto CommandList = IH.get_native_queue<backend::ext_oneapi_level_zero>();
+      auto SrcMem = IH.get_native_mem<backend::ext_oneapi_level_zero>(SrcA);
+      auto DstMem = IH.get_native_mem<backend::ext_oneapi_level_zero>(DstA);
+
+      // If L0 interop becomes a real use case we should make a new UR entry
+      // point to propagate events into and out of the the interop func.
```

Comment:
What does this comment mean?

Reply:
In order to make this entry point more performant on L0 we would need something like the previous [host task extensions](https://github.com/intel/llvm/pull/12921) so we can pass in and out events to the `ze` call. 

As it stands there is no point making this more performant for L0 since no one that we are aware of are using L0 interop in SYCL.